src/coreclr/vm/common.h

```diff
@@ -397,7 +397,7 @@ extern DummyGlobalContract ___contract;
 #endif
 
 void LogErrorToHost(const char* format, ...);
-
+#if defined(TARGET_ANDROID)
+void LogDebugToHost(const char* format, ...);
+#endif
```

Comment:
@elinor-fung What is the best way to express debug logging to the host?

Reply:
We only have an error writing callback (`coreclr_set_error_writer`) currently. Do we actually need debug logging to the host? The error logging to host is largely for runtime initialization issues, so the host can report them to the user. For debug logging once the runtime is actually going, I'd expect we could update the runtime logging mechanisms such that they use Android logging APIs instead.